packages/video_player/video_player_platform_interface/lib/video_player_platform_interface.dart

```diff
@@ -106,6 +113,24 @@ abstract class VideoPlayerPlatform extends PlatformInterface {
 
 class _PlaceholderImplementation extends VideoPlayerPlatform {}
 
+/// The video resolution, the number of distinct pixels in each dimension that
+/// can be displayed
```

Comment:
I think this can be improved as well:

```suggestion
/// The number of distinct pixels in each dimension that can be displayed for
///  a video.
```